### DISCUSSION TITLE
How to listen to accounts changed using @walletconnect/universal-provider?

### DISCUSSION THREAD

**Discussion by jingxuan98** [WalletConnect/walletconnect-monorepo#2844] [Developer Support]
How can I listen to accountchanged using @walletconnect/universal-provider?

1) Let say a user select another account in his app, how can I listen to that event and make changes in my frontend?

I realise theres no "accountsChanged" event listener for `universal-provider` like how` web3-provider` or `ethereum-provider` does, how can this be done for the `@walletconnect/universal-provider` ?

2) So as chain changed

**Comment by finessevanes** [comment]
@jingxuan98 actually the same events also exist: https://github.com/WalletConnect/walletconnect-monorepo/blob/35c16aeea0b3ac6b49f39f9ad24ae93eb130e7f7/providers/universal-provider/src/UniversalProvider.ts#L340
